.github/workflows/pr-pytest.yml

```diff
@@ -90,7 +90,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ['3.10']
+        python-version: ['3.10', '3.11-dev']
```

Comment:
I just added this temporarily to be able to check if the condition works in the PR - will remove it afterwards.